### PR TITLE
Make repo citable

### PR DESCRIPTION
**Pull request by janosh**
By adding `citation.cff`. Will show up in the side bar:

Screenshot 2023-09-16 at 12 08 51 PM

Sese https://github.com/materialsproject/pymatgen/commit/dfa4cdebdb1bc903dbd3958cffc1d194937ce877 for related `pymatgen` commit.

### CODE REVIEW

**Review by BowenD-UCB — APPROVED** [review]

File: citation.cff [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it as below.
+title: "CHGNet: Pretrained universal neural network potential for charge-informed atomistic modeling"
```
Comment (BowenD-UCB) [inline citation.cff:3]: The title of CHGNet on NMI has been changed to CHGNet as a pretrained universal neural network potential for charge-informed atomistic modelling

File: citation.cff [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - family-names: Han
+    given-names: Kevin
+  - family-names: Bartel
+    given-names: Christopher J.
+  - family-names: Ceder
+    given-names: Gerbrand
+date-released: 2023-02-24
+repository-code: https://github.com/CederGroupHub/chgnet
+arxiv: https://arxiv.org/abs/2302.14231
+doi: 10.1016/j.commatsci.2012.10.028
```
Comment (BowenD-UCB) [inline citation.cff:22]: This DOI is not CHGNet?

Comment (BowenD-UCB) [reply citation.cff:22]: https://doi.org/10.1038/s42256-023-00716-3

Comment (janosh) [reply citation.cff:22]: oops, copy-paste error. good thing you caught!

**Pull request merged**